server/database.py

```diff
@@ -339,9 +341,9 @@ def get(cls, id, *args, **kwargs):
             character = Character(
                 key=Key(Character, id),
                 user_id=id,
-                name=character_data['name'],
-                is_male=character_data['gender'] == 'male',
-                corporation_id=character_data['corporation_id'],
+                name=character_data.get('name', f'unknown {id}'),
```

Comment:
These defaults should not be here. If those values aren't present, it means the ESI endpoint had an error and we should be propagating it instead of creating a Character with essentially blank values that will never get replaced. For now we can leave the unhandled "key not found" exception. Opening an issue may be a good idea to track this.

Reply:
If I revert then Mail API will just crash.
Raised ticket #49
I'll just make it blank for now so it can run